x-pack/platform/plugins/shared/stack_connectors/server/connector_types/gemini/gemini.ts

```diff
@@ -286,6 +282,7 @@ export class GeminiConnector extends SubActionConnector<Config, Secrets> {
 
   public async invokeAI(
     {
+      maxOutputTokens,
```

Comment:
super duper nit: since this `maxOutputTokens` is optional, suggestion to move it below toolConfig